src/_posts/databases/mongodb/2000-01-01-compass.md

```diff
@@ -41,28 +41,29 @@ Fill the fields accordingly:
 {% assign img_url = "https://cdn.scalingo.com/documentation/compass/new_connection.png" %}
 {% include mdl_img.html %}
 
-The "SSH Tunnel" section must "Use Identity File". The identity file to provide
+The `SSH Tunnel` section must `Use Identity File`. The identity file to provide
```

Comment:
I wouldn't write "SSH Tunnel" and "Use Identity File" as a code, it's not a code at all 🤔